src/main/java/appeng/parts/p2p/FEP2PTunnelPart.java

```diff
@@ -172,9 +172,15 @@ public int getEnergyStored() {
     }
 
     private class OutputEnergyStorage implements IEnergyStorage {
+        private IEnergyStorage getInputStorage() {
```

Comment:
This is something we have to remove or not add as it can break worlds. It's is mostly trivial to add it in 1.17 when we can make a breaking change and apply it to every tunnel and not have to have custom handling for each and every one.